src/Sulu/Bundle/PageBundle/Resources/translations/admin.en.json

```diff
@@ -35,11 +35,11 @@
     "sulu_page.edited": "edited",
     "sulu_page.search_result": "Search Engine Result Pages Preview (SERP)",
     "sulu_page.meta_title": "Title (Meta title)",
-    "sulu_page.meta_title_info_text": "The meta title is limited to 55 characters.",
+    "sulu_page.meta_title_info_text": "The meta title is recommended to use 55 characters at most.",
```

Comment:
```suggestion
    "sulu_page.meta_title_info_text": "The meta title should be no longer than 55 characters.",
```

Reply:
Personally I would recommend to mention the seo guidelines in the info-text, something like `Seo guidelines recommend a title of 55 characters maximum.` So that it is clear that this is not only a recommendation from our side.

Reply:
```suggestion
    "sulu_page.meta_title_info_text": "According to the seo guidelines, the meta title should be no longer than 55 characters.",
```

Reply:
I think I would go with the word `recommended` here, other things sounds more like a hard limit from my side.

Reply:
pushed a new proposal - what do you think? 🙂

Reply:
looks good to me